src/platform/forms-system/src/js/helpers.js

```diff
@@ -235,7 +239,7 @@ export function stringifyFormReplacer(key, value) {
 
     // Exclude file data
     if (value.confirmationCode && value.file) {
-      return _.omit('file', value);
+      return omit('file', value);
```

Comment:
`omit` is fine. We could destructure like `{ file, ...remainingValue} = value;`, but then you'd have to do a lot more refactoring which would be messy. Plus the linter would complain about `file` not being used.